src/System.Diagnostics.PerformanceCounter/ref/System.Diagnostics.PerformanceCounter.cs

```diff
@@ -30,7 +30,7 @@ public partial struct CounterSample
         public static bool operator ==(System.Diagnostics.CounterSample a, System.Diagnostics.CounterSample b) { throw null; }
         public static bool operator !=(System.Diagnostics.CounterSample a, System.Diagnostics.CounterSample b) { throw null; }
     }
-    public sealed partial class PerformanceCounter
+    public sealed partial class PerformanceCounter : System.ComponentModel.Component, System.ComponentModel.ISupportInitialize
```

Comment:
Correct! I don't think that we use or ship it anywhere. I think it's more a placeholder for a cross-plat implementation. @joperezr do you have more insights?